### ISSUE TITLE
sensuctl configure non-interactive: config-dir flag is ignored

### ISSUE THREAD

**Issue by Surian** [sensu/sensu-go#2731]
## Expected Behavior
We should be able to use sensuctl configure with --non-interactive and --config-dir flags together.

## Current Behavior
When configuring sensuctl with sensuctl configure --non-interactive, --config-dir flag is ignored and config is always generated in ~/.config/sensu/sensuctl.
This doesn't happen when using interactive mode where --config-dir flag is correctly used.

## Steps to Reproduce (for bugs)

1. Execute
```
sensuctl configure --non-interactive \
    --config-dir /tmp/sensuctl \
    --url "api_url" \
    --username "admin" \
    --password "P@ssw0rd!" \
    --namespace default \
    --format yaml
```
2. You can see that configuration has been generated in $HOME/.config/sensu/sensuctl instead of  /tmp/sensuctl

## Context
We need to be able to generate multiple sensuctl configurations because you can have more than one sensu cluster to manage.

## Your Environment

* Sensu version used (sensuctl, sensu-backend, and/or sensu-agent): 5.2.1
* Installation method (packages, binaries, docker etc.): Ubuntu packages
* Operating System and version (e.g. Ubuntu 14.04): Ubuntu 18.04

**Comment by annaplotkin** [comment]
@ccressent to review

**Comment by ccressent** [comment]
I can confirm that this is related to #2345 and that the fix for #2345 will very likely take care of this issue too!
